src/pykeen/models/unimodal/node_piece.py

```diff
@@ -151,9 +158,21 @@ def __init__(
         relation_representations = embedding_specification.make(
             num_embeddings=2 * triples_factory.real_num_relations + 1,
         )
+        # pre-resolve tokenizer to select token representations
+        tokenizer = tokenizer_resolver.lookup(tokenizer)
+        tokenizer_cls = tokenizer.__class__ if isinstance(tokenizer, Tokenizer) else tokenizer
```

Comment:
Doesn't lookup() return the tokenizer class anyway?

Reply:
Nevermind, guess not.

Reply:
`lookup` passes through instances